src/Tests/Tests.csproj

```diff
@@ -109,9 +109,27 @@
     <None Include="app.config">
       <SubType>Designer</SubType>
     </None>
+    <None Include="differentialMultiFilesBackup1.bak">

```

Comment:
Can you possibly move all the bak files to a folder? to hide them away in Solutions Explorer

Reply:
Done.